hibp.js

```diff
@@ -58,7 +58,19 @@ const HIBP = {
   },
 
   async getUnsafeBreachesForEmail(sha1, allBreaches) {
-    return await this.getBreachesForEmail(sha1, allBreaches, true);
+    const allFoundBreaches = await this.getBreachesForEmail(sha1, allBreaches, true);
+
+    const breachLists = {
+      "websiteBreaches": allFoundBreaches.filter(breach => breach.IsSpamList === false),
+      "spamLists": allFoundBreaches.filter(breach => breach.IsSpamList === true),
```

Comment:
Not sure if using `.filter(breach => !breach.IsSpamList)` instead of `===` is any more efficient, or if there is a way to do this without using **two** loops for a very similar Boolean check. Possibly use `.reduce()` at the expense of a bit of readability.

Reply:
I like the readability > optimization. Programmer brain-time is more expensive than CPU clock-time.